### PR TITLE
add a new blog 

### PR DESCRIPTION
**Pull request by husiyu**
tidb-ecosystem-tools-2.md ( lightning introduction )

### CODE REVIEW

**Comment by husiyu** [comment]
@kennytm PTAL

File: tidb-ecosystem-tools-2.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+![](https://upload-images.jianshu.io/upload_images/542677-58fe4bed7f0e18a4.png?imageMogr2/auto-orient/strip%7CimageView2/2/w/1240)
+
+Lightning 现时只支持经 mydumper 导出的 SQL 备份。mydumper 将每个表的内容分别储存到不同的文件，与 mysqldump 不同。这样不用解析整个数据库就能平行处理每个表。
+
+首先，Lightning 会扫瞄 SQL 备份，区分出结构文件（包含 CREATE TABLE 语句）和数据文件（包含 INSERT 语句）。结构文件的内容会直接发送到 TiDB，用以建立数据库鶵型。
+
+然后 Lightning 就会并发处理每一张表的数据。这里我们只集中看一张表的流程。每个数据文件的内容都是规律的 INSERT 语句，像是：
+
+INSERT INTO `tbl` VALUES (1, 2, 3), (4, 5, 6), (7, 8, 9);  INSERT INTO `tbl` VALUES (10, 11, 12), (13, 14, 15), (16, 17, 18);  INSERT INTO `tbl` VALUES (19, 20, 21), (22, 23, 24), (25, 26, 27);
```
Comment (kennytm) [inline tidb-ecosystem-tools-2.md]:
``````suggestion
```sql
INSERT INTO `tbl` VALUES (1, 2, 3), (4, 5, 6), (7, 8, 9);  
INSERT INTO `tbl` VALUES (10, 11, 12), (13, 14, 15), (16, 17, 18);
INSERT INTO `tbl` VALUES (19, 20, 21), (22, 23, 24), (25, 26, 27);
```
``````

File: tidb-ecosystem-tools-2.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+Lightning 会作初步分析，找出每行在文件的位置并分配一个行号，使得没有主键的表可以唯一的区分每一行。此外亦同时将文件分割为大小差不多的区块（默认 256 MiB）。这些区块也会并发处理，让数据量大的表也能快速导入。以下的例子把文件以 20 字节为限分割成 5 块：
+
+INSERT INTO `tbl` VALUES (1, 2, 3), (4, 5, 6), (7, 8, 9);  INSERT INTO `tbl` VALUES (10, 11, 12), (13, 14, 15), (16, 17, 18);  INSERT INTO `tbl` VALUES (19, 20, 21), (22, 23, 24), (25, 26, 27);
```
Comment (kennytm) [inline tidb-ecosystem-tools-2.md]: There should be highlight of the selected rows here... maybe use an image.

**Comment by YiniXu9506** [comment]
LGTM

**Pull request merged**